nix/tenzir/plugins/names.nix

```diff
@@ -1,4 +1,5 @@
 [
+  "azure-log-analytics"
```

Comment:
This also needs to be added in `Dockerfile` and `.github/workflows/tenzir.yaml`.

Reply:
Oh, right. Done in https://github.com/tenzir/tenzir/pull/4281/commits/a892402f6b323d4844979a678847242486f5f4b4.